### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by manuelvicnt**

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## :loudspeaker: Type of change
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix
+- [ ] New feature
+- [ ] Refactoring
+
+## :bulb: Description
+
```
Comment (keyboardsurfer) [inline pull_request_template.md]: Nit: To be consistent, add another new line here.

File: pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+## :scroll: Changes
+
+
+## :green_heart: How did you test it?
+
+
+## :pencil: Checklist
+_Put an `x` in the boxes that apply_
+
+- [ ] I run `./gradlew spotlessApply` before submitting the PR
```
Comment (keyboardsurfer) [inline pull_request_template.md]: Nit: "Ran `..."

File: pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] I have performed a self-review of my own code
```
Comment (keyboardsurfer) [inline pull_request_template.md]: Nit: "Reviewed submitted code"

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (keyboardsurfer) [inline pull_request_template.md]: Nit: "Added tests to verify changes"

**Comment by ashdavies** [comment]
You could also add a comment on pairing, this is a snippet from our repo
```
## 👥 Pairing
<!-- Did you collaborate with a co-author? Make sure to attribute them! -->
https://github.com/blog/2496-commit-together-with-co-authors
```

**Comment by manuelvicnt** [comment]
Thanks for the idea @ashdavies ! I think we can add that on demand for the time being. Wish we could do more pairing but the reality is that we don't do it that often.

**Pull request merged**